Description: Ensure gdm has an ubuntu styling by default.
 As it's system-wide and not user-wide, use the ubuntu.css stylesheet
 for the gdm mode of GNOME Shell. This doesn't impact the aNOME
 user session though.
Forwarded: not-needed
Author: Didier Roche <didrocks@ubuntu.com>
Index: gnome-shell-3.24.3/js/ui/sessionMode.js
===================================================================
--- gnome-shell-3.24.3.orig/js/ui/sessionMode.js
+++ gnome-shell-3.24.3/js/ui/sessionMode.js
@@ -43,6 +43,7 @@ const _modes = {
 
     'gdm': {
         hasNotifications: true,
+        stylesheetName: 'ubuntu.css',
         isGreeter: true,
         isPrimary: true,
         unlockDialog: imports.gdm.loginDialog.LoginDialog,
